.buildkite/common.py

```diff
@@ -248,6 +248,12 @@ def __init__(self, initial_steps=None, **kwargs):
         step_build = group("🏗️ Build", build_cmds, **self.per_arch)
         self.steps += [step_build, "wait"]
 
+        if initial_steps:
+            for step in initial_steps:
+                step["depends_on"] = None
```

Comment:
nice! Can we add a short description here. Otherwise I am sure I will forget.

Reply:
Hahaha, sure thing, I'll add it as part of https://github.com/firecracker-microvm/firecracker/pull/4660, if that's okay? since we already have 2 approvals here :)